std_msgs/Header header
Distribution[]  data
